aptos-move/framework/aptos-framework/sources/configs/consensus_config.spec.move

```diff
@@ -40,6 +43,9 @@ spec aptos_framework::consensus_config {
         requires timestamp::spec_now_microseconds() >= reconfiguration::last_reconfiguration_time();
         requires exists<stake::ValidatorFees>(@aptos_framework);
         requires exists<CoinInfo<AptosCoin>>(@aptos_framework);
+
+        ensures @aptos_framework == addr;
```

Comment:
Same, since it is already described as an aborts_if condition, there is no need to add an ensures on a value depending on the pre-state of the function. Please remove it

Reply:
fixed